src/platform/Ameba/Logging.cpp

```diff
@@ -32,9 +32,9 @@
 
 namespace chip {
 namespace Logging {
-namespace Platform {
+namespace Ameba {
 
-static uint8_t LogLevel = AmebaLogLevel::ameba_loglevel_detail;
+static AmebaLogLevel LogLevel = AmebaLogLevel::kDetail;
```

Comment:
nit: prefer anonymous namespace instead of `static`

Reply:
fixed